flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerMigrationOperatorTest.java

```diff
@@ -28,6 +28,12 @@
 
 /**
  * Migration test from FlinkKafkaProducer011 operator.
+ * This test depends on the resource generated by {@link FlinkKafkaProducer011MigrationTest#writeSnapshot()}.
+ *
+ * <p>Warning: We need to rename the generated resource based on the file naming pattern specified by
+ * the {@link #getOperatorSnapshotPath(MigrationVersion)} method then
+ * copy the resource to the path also specified by the {@link #getOperatorSnapshotPath(MigrationVersion)} method.
+ * </p>
```

Comment:
Javadoc shouldn't have a closing `</p>` tag.

Reply:
Thanks for your review. I have removed `</p>` tag.